client/knode/validator/validator.go

```diff
@@ -625,7 +630,7 @@ func (val *validator) AddBlockFragment(blockNumber *big.Int, round uint64, fragm
 		err = val.chain.Validator().ValidateState(block, parent, val.state, receipts, usedGas)
 		if err != nil {
 			val.handleMutex.Unlock()
-			log.Crit("Failed to validate the state", "err", err)
+			log.Error("Failed to validate the state", "err", err)
```

Comment:
I'm worried about all those `Crit` to `Error`. I guess they were `Crit` because the state of the current process is invalid so we have to crash the process. Moving it to error means the process will continue. Is it fine? Will the state get fixed?